docroot/modules/humsci/hs_views_helper/config/optional/views.view.data_export.yml

```diff
@@ -0,0 +1,1516 @@
+          element_default_classes: true
+          empty: ''
+          hide_empty: false
+          empty_zero: false
+          hide_alter_empty: true
+          click_sort_column: target_id
+          type: entity_reference_label
+          settings:
+            link: false
+          group_column: target_id
+          group_columns: {  }
+          group_rows: true
+          delta_limit: 0
+          delta_offset: 0
+          delta_reversed: false
+          delta_first_last: false
+          multi_type: separator
+          separator: ', '
+          field_api_classes: false
+          plugin_id: field
+        uri:
+          id: uri
+          table: file_managed
+          field: uri
+          relationship: field_media_image_target_id
+          group_type: group
+          admin_label: 'Field: Image'
+          label: ''
+          exclude: false
+          alter:
+            alter_text: false
+            text: ''
+            make_link: false
+            path: ''
+            absolute: false
+            external: false
+            replace_spaces: false
+            path_case: none
+            trim_whitespace: false
+            alt: ''
+            rel: ''
+            link_class: ''
+            prefix: ''
+            suffix: ''
+            target: ''
+            nl2br: false
+            max_length: 0
+            word_boundary: true
+            ellipsis: true
+            more_link: false
+            more_link_text: ''
+            more_link_path: ''
+            strip_tags: false
+            trim: false
+            preserve_tags: ''
+            html: false
+          element_type: ''
+          element_class: ''
+          element_label_type: ''
+          element_label_class: ''
+          element_label_colon: false
+          element_wrapper_type: ''
+          element_wrapper_class: ''
+          element_default_classes: true
+          empty: ''
+          hide_empty: false
+          empty_zero: false
+          hide_alter_empty: true
+          click_sort_column: value
+          type: file_uri
+          settings:
+            link_to_file: false
+            file_download_path: true
+          group_column: value
+          group_columns: {  }
+          group_rows: true
+          delta_limit: 0
+          delta_offset: 0
+          delta_reversed: false
+          delta_first_last: false
+          multi_type: separator
+          separator: ', '
+          field_api_classes: false
+          entity_type: file
+          entity_field: uri
+          plugin_id: field
+        field_s_event_speaker:
+          id: field_s_event_speaker
+          table: node__field_s_event_speaker
+          field: field_s_event_speaker
+          relationship: none
+          group_type: group
+          admin_label: ''
+          label: ''
+          exclude: false
+          alter:
+            alter_text: true
+            text: '<div data-attribute-tag="person">{{ field_s_event_speaker }}</div>'
```

Comment:
What is this data property for?

Reply:
https://github.com/SU-HSDO/suhumsci/pull/160/files#diff-aa13bdef96022fd9205ffe1d82cbe38bR37

its the only way i could find to have nested xml tags

Reply:
Hmmm. Yeah, this all seems to work and I see why you went this route. I tried to swap out for a new view mode but the template output was not great. I am not sure why the only formatter is `bricks` and why it only has one default mode. This seems like something that could have been expanded on but might not be worth the effort. Nice work on the custom root and item tags tho.

Reply:
even with different view modes its still renders out just plain html. so even with paragraphs/eck its the same outcome.